src/App.js

```diff
@@ -1,25 +1,16 @@
-import logo from './logo.svg';
-import './App.css';
+import './index.css';
+import { Component } from 'react';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+// eslint-disable-next-line react/prefer-stateless-function
+class App extends Component {
+  render() {
+    return (
+      <>
+        <Calculator />
+      </>
+    );
+  }
 }
```

Comment:
### OPTIONAL 

- [ ] It would be better for your App.js to be a functional component, functional components are the modern ways to write  components 👍🏿